src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/overview/overview.component.html

```diff
@@ -1,34 +1,43 @@
-<div class="row">
-  <div class="col-md-12">
+<ng-container *ngIf="orchAvailable$ | async as orchAvailable; else mirroringDetailsTpl">
+  <ng-container *ngIf="!rbdPoolExists || !(rbdServiceExists$ | async); else mirroringDetailsTpl">
+    <cd-empty-state btnText="Configure RBD Mirroring"
+                    msg="RBD mirroring is not configured."
+                    helperText="This will create rbd-mirror service and a replicated RBD pool"
```

Comment:
I added i18n in their respective areas. Here its just variable names where I shouldn't do this.

Reply:
The problem is that i18n needs literal strings. It's not smart enough to guess which future value a variable will hold. So you have to tag where the literal is added. If not, translators will be presented with translation strings as `"{helperText}"`, instead of `"This will create rbd-mirror service and a replicated RBD pool"`.